canonical-web-and-design/snapcraft.io
# Use ModularTable for builds

## Done

Update builds table to use new `ModularTable` react compoenent.

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/ or [demo](https://snapcraft-io-3063.demos.haus)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check builds table of any snap, make sure it renders correctly

## Screenshots

<img width="1270" alt="Screenshot 2020-09-21 at 13 11 51" src="https://user-images.githubusercontent.com/83575/93760419-44111a00-fc0c-11ea-86eb-e5fad368fc5e.png">
